streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamPrint.java

```diff
@@ -30,15 +31,15 @@ public KStreamPrint(final ForeachAction<K, V> action) {
     }
 
     @Override
-    public Processor<K, V> get() {
+    public Processor<K, V, K, V> get() {
         return new KStreamPrintProcessor();
     }
 
-    private class KStreamPrintProcessor extends AbstractProcessor<K, V> {
+    private class KStreamPrintProcessor extends AbstractProcessor<K, V, K, V> {
```

Comment:
```suggestion
    private class KStreamPrintProcessor extends Processor<K, V, K, V> {
```
See my comment on AbstractProcessor. Since we don't need the context, we don't need to depend on any abstract class (since the new Processor has defaults for init and close).